website/settings/defaults.py

```diff
@@ -1831,4 +1831,5 @@ def parent_dir(path):
     'bluebook-inline': 'bluebook'
 }
 
-PREPRINTS_ASSETS = '/static/img/preprints_assets/'
+OSF_CDN_DOMAIN = 'https://staging-cdn.osf.io'
```

Comment:
@steve @brianjgeiger 
For local and staging, use https://staging-cdn.osf.io.
For prod and test, https://cdn.osf.io seems not ready?